src/app/tests/archieve_license_namespace_views_test.py

```diff
@@ -0,0 +1,98 @@
+# -*- coding: utf-8 -*-
+
+
+from django.urls import reverse
+from django.contrib.staticfiles.testing import StaticLiveServerTestCase
+
+from selenium import webdriver
+from selenium.webdriver.firefox.options import Options
+from webdriver_manager.firefox import GeckoDriverManager
+
+from app.models import LicenseNamespace
+from app.generateXml import generateLicenseXml
+
+
+
+class ArchiveLicenseNamespaceViewsTestCase(StaticLiveServerTestCase):
+
+    def setUp(self):
```

Comment:
Let's have a common setup and teardown function in `test_utils.py` and use it wherever we can.